.github/scripts/test_kilosort4_ci.py

```diff
@@ -537,33 +561,16 @@ def monkeypatch_filter_function(self, X, ops=None, ibatch=None):
         # memory file. Because in this test recordings are preprocessed, there are
         # some filter edge effects that depend on the chunking in `get_traces()`.
         # These are all extremely close (usually just 1 spike, 1 idx different).
-        results = self._get_sorting_output(kilosort_output_dir, spikeinterface_output_dir)
+        results = {}
+        results["ks"] = {}
+        results["ks"]["st"] = np.load(kilosort_output_dir / "spike_times.npy")
+        results["ks"]["clus"] = np.load(kilosort_output_dir / "spike_clusters.npy")
+        results["si"] = {}
+        results["si"]["st"] = np.load(spikeinterface_output_dir / "sorter_output" / "spike_times.npy")
+        results["si"]["clus"] = np.load(spikeinterface_output_dir / "sorter_output" / "spike_clusters.npy")
         assert np.allclose(results["ks"]["st"], results["si"]["st"], rtol=0, atol=1)
```

Comment:
Should we also check `results["ks"]["clus"] == results["si"]["clus"]` here?

Reply:
done